wordpress-mobile/WordPress-iOS
# Fix help button crash

Fixes #8199

To test:
- ensure the help button works on the Google login error screen
- also the new site creation screen

Needs review: @ScoutHarris
